### PR TITLE
docs: adding project board to read me

### PR DESCRIPTION
**Pull request by makyfj**
### 🛠️ Fixes Issue
Adding project board based on this issue [Issue](https://github.com/ykdojo/defaang/issues/97)

### 👨‍💻 Changes proposed

Added link for project board on readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,10 @@ Currently planning to use:
 - backend: Supabase
 - hosting: Vercel
 
+## Project Board - Road Map
```
Comment (ykdojo) [inline README.md:20]: Road Map -> roadmap

Project Board -> Project board

**Review by ykdojo — CHANGES_REQUESTED** [review]
chore -> docs

Thank you!

File: README.md (hunk 1 above)
Comment (makyfj) [reply README.md]: Got it :D

**Review by ykdojo — APPROVED** [review]

**Review by RakeshSangem — APPROVED** [review]
Looking good to me🎉

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Readable

**Review by rohitdasu — APPROVED** [review]

**Pull request merged**